CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ PLEASE REMEMBER TO CHANGE THE '..main' WITH AN ACTUAL TAG in GITHUB LINK.
 
 ## jax 0.2.17 (unreleased)
 * [GitHub commits](https://github.com/google/jax/compare/jax-v0.2.16...main).
+* New features:
+  * New SciPy function {py:func} `jax.scipy.special.sph_harm`.
```

Comment:
Here and below, I think we need to remove the space after `{py:func}` for it to render correctly

Reply:
Thanks! Done.